tests/disaster-recovery/sc_arbiter/conftest.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.resources.deployment import Deployment
+from ocs_ci.ocs.resources.job import get_job_obj
+from ocs_ci.ocs.resources.pod import (
+    get_pods_having_label,
+    wait_for_pods_to_be_running,
+    get_deployments_having_label,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(items):
+    """
+    A pytest hook to  filter out SC Arbiter tests
```

Comment:
if you are going to send another commit, please replace `SC` with `Stretched cluster`